features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha_alt_hw.c

```diff
@@ -318,17 +317,17 @@ void crypto_sha_update_nobuf(crypto_sha_context *ctx, const unsigned char *input
     // 1. Last block which may be incomplete
     // 2. Non-last block which is complete
     MBED_ASSERT(islast || ilen == ctx->blocksize);
-    
+
     const unsigned char *in_pos = input;
     int rmn = ilen;
     uint32_t sha_ctl_start = (CRPT->HMAC_CTL & ~(CRPT_HMAC_CTL_DMALAST_Msk | CRPT_HMAC_CTL_DMAEN_Msk | CRPT_HMAC_CTL_HMACEN_Msk)) | CRPT_HMAC_CTL_START_Msk;
     uint32_t sha_opmode = (CRPT->HMAC_CTL & CRPT_HMAC_CTL_OPMODE_Msk) >> CRPT_HMAC_CTL_OPMODE_Pos;
     uint32_t DGST0_old = 0, DGST1_old = 0, DGST2_old = 0, DGST3_old = 0, DGST4_old = 0, DGST5_old = 0, DGST6_old = 0, DGST7_old = 0,
```

Comment:
minor: code would have looked tidy with an array instead of so many similar looking variables. Just like ```HMAC_DGST[]```.

Reply:
@mazimkhan I refine the code in https://github.com/ARMmbed/mbed-os/pull/4925/commits/23a81d3099dce884f575eb2a59d9eed4dc02f6fc.